src/main/java/de/hysky/skyblocker/skyblock/museum/DonationButton.java

```diff
@@ -0,0 +1,174 @@
+package de.hysky.skyblocker.skyblock.museum;
+
+import de.hysky.skyblocker.skyblock.item.WikiLookup;
+import de.hysky.skyblocker.skyblock.itemlist.ItemRepository;
+import de.hysky.skyblocker.utils.ItemUtils;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.font.TextRenderer;
+import net.minecraft.client.gui.DrawContext;
+import net.minecraft.client.gui.screen.narration.NarrationMessageBuilder;
+import net.minecraft.client.gui.screen.recipebook.AnimatedResultButton;
+import net.minecraft.client.gui.widget.ClickableWidget;
+import net.minecraft.client.render.RenderLayer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.screen.ScreenTexts;
+import net.minecraft.text.MutableText;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import net.minecraft.util.Pair;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+public class DonationButton extends ClickableWidget {
+	private static final int SIZE = 33;
+	private static final int ITEM_OFFSET = 8;
+
+	private Donation donation = null;
+	private ItemStack itemStack = null;
+	private static final String WIKI_LOCKUP_KEY = WikiLookup.wikiLookup.getBoundKeyTranslationKey();
```

Comment:
If this value can change (e.g. due to the key being rebound) then it shouldn't be saved as a constant, if it can't then its fine as it is.